src/components/PokemonList.jsx

```diff
@@ -12,17 +12,26 @@ const PokemonList = ({
 }) => {
   const [pokemons, setPokemons] = useState([]);
   const [query, setQuery] = useState('');
-  const handleQueryChange = (e) => {
-    setQuery(e.target.value);
-  };
+  const [offset, setOffset] = useState(0);
 
   useEffect(() => {
     axios
-      .get('https://pokeapi.co/api/v2/pokemon?limit=100&offset=0')
+      .get(`https://pokeapi.co/api/v2/pokemon?limit=100&offset=${offset}`)
       .then(({ data }) => {
         setPokemons(data.results);
       });
-  }, []);
+  }, [offset]);
+
+  const handleQueryChange = (e) => {
+    setQuery(e.target.value);
+  };
+
+  const handleList = (e) => {
+    const choice = e.target.value;
+
+    setQuery('');
+    setOffset(parseInt(choice.slice(0, choice.indexOf('-')), 10) - 1);
```

Comment:
This bit of code is overkill, just use the `value` attribute of your `option` !

Reply:
👍